subsys/net/lib/lwm2m_client_utils/lwm2m/lwm2m_security.c

```diff
@@ -446,25 +450,25 @@ static struct settings_handler lwm2m_security_settings = {
 	.h_commit = loaded,
 };
 
-static int write_to_settings(int obj, int inst, int res, uint8_t *data, uint16_t data_len)
+static int write_to_settings(uint16_t obj, uint16_t inst, uint16_t res, uint8_t *data, uint16_t data_len)
 {
-	char path[sizeof(SETTINGS_PREFIX "/0/0/10")];
-
-	snprintk(path, sizeof(path), SETTINGS_PREFIX "/%d/%d/%d", obj, inst, res);
-	if (settings_save_one(path, data, data_len)) {
-		LOG_ERR("Failed to store %s", path);
-	}
-	LOG_DBG("Permanently stored %s", path);
-	return 0;
+    char path[SETTINGS_PATH_LEN];
+
+    snprintk(path, sizeof(path), SETTINGS_PATH_FMT, obj, inst, res);
+    if (settings_save_one(path, data, data_len)) {
+        LOG_ERR("Failed to store %s", path);
+    }
+    LOG_DBG("Permanently stored %s", path);
+    return 0;
 }
 
-static void delete_from_storage(int obj, int inst, int res)
+static void delete_from_storage(uint16_t obj, uint16_t inst, uint16_t res)
 {
-	char path[sizeof(SETTINGS_PREFIX "/0/0/0")];
+    char path[SETTINGS_PATH_LEN];
 
-	snprintk(path, sizeof(path), SETTINGS_PREFIX "/%d/%d/%d", obj, inst, res);
-	settings_delete(path);
-	LOG_DBG("Deleted %s", path);
+    snprintk(path, sizeof(path), SETTINGS_PATH_FMT, obj, inst, res);
+    settings_delete(path);
+    LOG_DBG("Deleted %s", path);
```

Comment:
Tab length seems to be incorrectly 4 here.

Reply:
OK i will fix